plugins/out_prometheus_remote_write/remote_write.h

```diff
@@ -26,12 +26,26 @@
 #define FLB_PROMETHEUS_REMOTE_WRITE_MIME_PROTOBUF_LITERAL    "application/x-protobuf"
 #define FLB_PROMETHEUS_REMOTE_WRITE_VERSION_HEADER_NAME      "X-Prometheus-Remote-Write-Version"
 #define FLB_PROMETHEUS_REMOTE_WRITE_VERSION_LITERAL          "0.1.0"
+#ifdef FLB_HAVE_SIGNV4
+#ifdef FLB_HAVE_AWS
+#define FLB_HTTP_AWS_CREDENTIAL_PREFIX "aws_"
```

Comment:
Please update this to `FLB_PROMETHEUS_REMOTE_WRITE_CREDENTIAL_PREFIX`

Reply:
ok